bitnami/cert-manager/templates/_helpers.tpl

```diff
@@ -150,12 +150,12 @@ Compile all warnings into a single message.
 {{- end -}}
 {{- end -}}
 
-{{/* Validate values of Cert Manager - CRD */}}
+{{/* Validate values of cert-manager - CRD */}}
```

Comment:
I see changes in a lot of charts, please fix them

Reply:
Solved [aea8dc8](https://github.com/bitnami/charts/pull/13480/commits/aea8dc8eb9a80399a557b791cbe14c0f4012447b)